src_py/apiServer/transmitter.py

```diff
@@ -93,9 +93,7 @@ def train(self):
         globe.experiment_flow_global.syncTrainingWithFlow()
 
         # 1 Ack for clientsTraining(), <num of sources> Acks for updateCSV():
-        globe.pendingAcks += 1 + len(globe.components.sources) 
-
-        self.updateCSV(globe.TRAINING_STR)
+        globe.pendingAcks += 1
```

Comment:
Please write here a short explanation about what was happened here before

Why did you remove "+ len(globe.components.sources)"

Reply:
moved functionability to "sendData" function